addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -1,3 +1,14 @@
+// Temporary CSS rule, while the rules located in 
+// 'addons/web/static/src/webclient/icons.scss' are moved to another location.
+// Currently, the rules in this file have no effect on this snippet.
+
+.o_rtl .s_image_gallery[data-vcss="001"] {
+    // Flip directional icons by 180 degree.
+    .oi-chevron-left,
+    .oi-chevron-right {
+        transform: rotate(180deg);
+    }
+}
```

Comment:
this diff

Reply:
@rdeodoo Ok. So we wait for the design task, and I remove this snippet of code. That'll correct itself.

Reply:
Something like that I guess.

If you want to already do your commit as if the design task was done, to fix the remaining things, feel free :+1: 
(could even be merged before the design fix then)

Would need to adapt the commit to explain what's fixed now that you removed the diff about the snippet, and to include a link and explanation to the design task in the commit msg :+1:

Reply:
@rdeodoo I've adapted the commit with a link to the design task and an explanation about it.